experiments/AMIP/coupler_driver.jl

```diff
@@ -77,6 +77,7 @@ include("components/ocean/eisenman_seaice.jl")
 ## helpers for user-specified IO
 include("user_io/user_diagnostics.jl")
 include("user_io/user_logging.jl")
+include("user_io/debug_plots.jl")
```

Comment:
Do we want to include this in main, or just output debug plots for interactive runs like we do now?

Reply:
I think it's fine to output them! Just wanted to check that it was intentional to leave this in